### ISSUE TITLE
Not narrating accessible name of not focusable row via aria-activedescendant

### ISSUE THREAD

**Issue by adamsamec** [FreedomScientific/standards-support#808]
### Summary

With Google Chrome, when the aria-activedescendant attribute is used on an input with role="combobox" and refers to a  row inside a grid, the accessible name of the row is not narrated upon that combobox focus if the row is both not focusable and does not have either aria-label or aria-labelledby attribute. Instead, only "row"is narrated upon that combobox focus.

The example code is below:

    <input type="text" role="combobox" aria-label="Type here" aria-activedescendant="descendant-row1">
    <div role="grid">
        <div id="descendant-row1" role="row">
            <div role="gridcell">This is a descendant</div>
        </div>
    </div>

#### Steps to reproduce

1. Visit the [example page][example].
2 . Navigate to the "Start here" button using the Tab key.
3. Press the Tab key to navigate to the "Type here" combobox.
4. Listen to the JAWS narration.

### Expected result

JAWS should narrate: "This is a descendant. Row".

### Actual result

JAWS instead narrates just: "Row".

### Example

[Example is here][example].

#### JAWS version and build number

JAWS 2023.2307.37.
 
#### Operating System and version

Microsoft Windows 11.

#### Browser and version:

The issue has been observed in Google Chrome and Microsoft Edge. Mozilla Firefox does not exhibit the issue.

[example]: https://jsbin.com/balugonefu/edit?html,output

**Comment by stevefaulkner** [comment]
please review the accessibility tree in chrome, in the [first case](https://codepen.io/stevef/pen/KKErdZM) the row does not have an accessible name, thus will not be announced. This is not a JAWS bug.

![the row has an empty accessible name](https://github.com/FreedomScientific/standards-support/assets/835859/7ed89e64-980a-49a6-b89e-a9f4817a1b65)

**Comment by adamsamec** [comment]
Sorry, my bad. I have reported the [bug for Chromium #324786153](https://issues.chromium.org/issues/324786153)
